webapp/security/alembic/versions/8e9386e7a94b_add_component_column_to_status_table.py

```diff
@@ -0,0 +1,35 @@
+"""add component column to status table
+
+Revision ID: 8e9386e7a94b
+Revises: fc282ca1dc4d
+Create Date: 2020-07-16 15:01:39.739465
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "8e9386e7a94b"
+down_revision = "fc282ca1dc4d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    components = ["main", "universe"]
+    components_str = "'" + "','".join(components) + "'"
+    op.execute(f"CREATE TYPE components AS ENUM ({components_str});")
+    op.add_column(
+        "status",
+        sa.Column(
```

Comment:
Similarly here, I wonder if it's possible to create the column and say "set all existing items to the default value of 'main'" rather than them all populating with "NULL"?

Reply:
The database does not allow NULL for component and defaults to `main` if no component is provided. 
```python
28            nullable=False,
29            server_default="main",
```

I did not set it in the `module.py` though. Thinking that this should be enough for the right data to be stored in the db, but I will take a look.